public/docs/_examples/animations/ts/app/hero-list-basic.component.ts

```diff
@@ -0,0 +1,53 @@
+ // #docregion
+ // #docregion imports
+import {
+  Component,
+  Input,
+  trigger,
+  state,
+  style,
+  transition,
+  animate
+} from '@angular/core';
+// #enddocregion imports
+import { Hero, Heroes } from './hero.service';
+
+@Component({
+  moduleId: module.id,
+  selector: 'hero-list-basic',
+  // #docregion template
+  template: `
+    <ul>
+      <li *ngFor="let hero of heroes.get()"
+          @heroState="hero.state"
```

Comment:
Recommend you call this `@heroSelected` since this reflects (via styles) whether the hero is active/inactive (based on click selections).